src/main.rs

```diff
@@ -137,6 +138,7 @@ impl Bk<'_> {
             line: 0,
             mark: HashMap::new(),
             links: epub.links,
+            return_stack: Vec::with_capacity(5), 
```

Comment:
i don't like this magic number

Reply:
Ok. Should I change it to a constant, or should I change it to use just new Vec? 
The idea behind 5 was a soft limit for most cases.

Reply:
just new vec. there's no reason to try to optimize this